Cannot build on Debian 13

Firstly many of the dependency names are wrong:
kf6-extra-cmake-modules > extra-cmake-modules
kf6-coreaddons-dev > libkf6coreaddons-dev
kf6-declarative-dev > libkf6declarative-dev
kf6-kpackage-dev > libkf6package-dev
kf6-kdbusaddons-dev > libkf6dbusaddons-dev
kf6-kiconthemes-dev > libkf6iconthemes-dev
kf6-ksvg-dev > libkf6svg-dev
kf6-newstuff-dev > libkf6newstuff-dev
kio6-dev > libkf6kio-dev
kdoctools6-dev > libkf6doctools-dev

Secondly, attempting to actually build it returns this error:
`dh_auto_configure: error: cd obj-x86_64-linux-gnu && DEB_PYTHON_INSTALL_LAYOUT=deb PKG_CONFIG=/usr/bin/pkg-config cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=None -DCMAKE_INSTALL_SYSCONFDIR=/etc -DCMAKE_INSTALL_LOCALSTATEDIR=/var -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON -DFETCHCONTENT_FULLY_DISCONNECTED=ON -DCMAKE_INSTALL_RUNSTATEDIR=/run "-GUnix Makefiles" -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_INSTALL_LIBDIR=lib/x86_64-linux-gnu .. returned exit code 1
make: *** [debian/rules:4: binary] Error 2
dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2`

If you have found that the building works on Debian, I suggest that you upload a .deb package so that end users needn't build it.

--- a/README.md
+++ b/README.md
@@ -22,6 +22,7 @@ Irix, NetBSD, OpenBSD, Solaris, and Tru64 with varying degrees of completion.
 1) Install build dependencies:
 
 ```bash
+sudo apt install build-essential debhelper-compat
 sudo apt-get build-dep .
 ```
 

--- a/debian/control
+++ b/debian/control
@@ -3,28 +3,29 @@ Section: kde
 Priority: optional
 Maintainer: Volodymyr Zolotopupov <zvova7890@gmail.com>
 Rules-Requires-Root: no
+Standards-Version: 4.6.2
 Build-Depends:
- debhelper-compat (= 12),
+ debhelper-compat (= 13),
  cmake,
  quilt,
- kf6-extra-cmake-modules,
+ extra-cmake-modules | kf6-extra-cmake-modules,
  qt6-base-dev,
  qt6-declarative-dev,
- kf6-kcoreaddons-dev,
- kf6-kdeclarative-dev,
- kf6-kpackage-dev,
- kf6-kdbusaddons-dev,
- kf6-kiconthemes-dev,
- kf6-kitemviews-dev,
- kf6-ksvg-dev,
- kf6-knewstuff-dev,
+ libkf6coreaddons-dev | kf6-kcoreaddons-dev,
+ libkf6declarative-dev | kf6-kdeclarative-dev,
+ libkf6package-dev | kf6-kpackage-dev,
+ libkf6dbusaddons-dev | kf6-kdbusaddons-dev,
+ libkf6iconthemes-dev | kf6-kiconthemes-dev,
+ libkf6itemviews-dev | kf6-kitemviews-dev,
+ libkf6svg-dev | kf6-ksvg-dev,
+ libkf6newstuff-dev | kf6-knewstuff-dev,
  libkf6notifications-dev,
  libkf6windowsystem-dev,
  libkf6configwidgets-dev,
  libkf6globalaccel-dev,
  libkf6xmlgui-dev,
- kio6-dev,
- kdoctools6-dev,
+ libkf6kio-dev | kio6-dev,
+ kdoctools-dev | libkf6doctools-dev,
  libkf6auth-dev,
  libplasma-dev,
  libkf6kcmutils-dev,
@@ -32,34 +33,14 @@ Build-Depends:
  libksysguard-dev,
  qt6-webengine-dev,
  libpcap0.8-dev,
- libpcap-dev,
  libsensors-dev,
-Standards-Version: 4.5.0
+ libnl-3-dev,
+ libnl-genl-3-dev,
+ libnl-route-3-dev
 
 Package: ksysguard6
 Architecture: any
 Depends:
- kf6-extra-cmake-modules,
- qt6-base,
- qt6-declarative,
- kf6-kcoreaddons,
- kf6-kdeclarative,
- kf6-kpackage,
- kf6-kdbusaddons,
- kf6-kiconthemes,
- kf6-kitemviews,
- kf6-ksvg,
- kf6-knewstuff,
- libkf6notifications6,
- libkf6windowsystem6,
- libkf6configwidgets6,
- libkf6globalaccel6,
- libkf6xmlgui6,
- kio6,
- libkf6auth6,
- libkf6config-bin,
- libksysguard,
- qt6-webengine,
- libpcap0.8,
- lm-sensors,
-Description: Track and control the processes running in your system. KF6 version.
+ ${shlibs:Depends},
+ ${misc:Depends}
+Description: Track and control the processes running in your system (KF6 version)

